src/main/java/freiman/fallingsand/Sand.java

```diff
@@ -0,0 +1,56 @@
+package freiman.fallingsand;
+
+public class Sand {
+
+    private int [][] field = new int[3][3];
```

Comment:
I didn't ask this for HW but can you make the size of field configurable by passing a width and height into the constructor?